internal/statestore/redis_indices.go

```diff
@@ -72,6 +78,20 @@ func extractIndexFilters(p *pb.Pool) []indexFilter {
 		})
 	}
 
+	for _, f := range p.BoolEqualsFilters {
+		min := -0.5
+		max := 0.5
+		if f.Value {
```

Comment:
Redis returns all the elements in the sorted set at key with a score between max and min (including elements with score equal to max or min), would it be more clear if we use filter {0,0} for false and {1,1} for true?

Reply:
This feels more obviously correct to me.  As I didn't have to look up that behavior to know it would work.  How strong is this concern?

Reply:
Correctness wise, both are correct :). Readability wise, I think setting to 0, 1 and checking for {0,0} {1,1} seems more obvious to me. When i read this, i assumed that may be same value for bounds didnt work for redis filtering and so you ve used these other ranges. If they do - then I do think thats clearer.

Reply:
I ll merge this in interest of getting the API change sooner. You can address this in the related next PR?